File: volume 100/10018.cpp
#include<iostream>
using namespace std;
long long reverse_number(long long n){
    long long k=0;
    while(n>0){
        k*=10;
        k+=n%10;
        n/=10;
    }
    return k;
}
int main(){
    int n;
    cin>>n;
    while(n--){
        long long s;
        cin>>s;
        long long rev_s;
        rev_s=reverse_number(s);
        int cnt=0;
        while(true){
            s+=rev_s;
            cnt++;
            rev_s=reverse_number(s);
            if(rev_s==s) break;
        }
        
        cout<<cnt<<" "<<s<<endl;
    }
}

File: volume 100/10018.py
n=int(input())
while(n>0):
    n-=1
    s=str(input())
    cnt=0
    rev_s=s[::-1]
    while True:
        s=str(int(s)+int(rev_s))
        cnt+=1
        rev_s=s[::-1]
        if(rev_s==s): break
    print(cnt,end=" ")
    print(s)
